### ISSUE TITLE
Decreased amyloid beta 40 isoform level 

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7074]
**Preferred term label:**
Decreased amyloid beta 40 isoform level 

**Synonyms**
Decreased beta amyloid 40 level, Decreased Abeta 40 level

**Definition (free text, please give PubMed ID)**
Lower than normal level of amyloid beta 40 found in cerebrospinal fluid.
PMID:31391004

**Parent term (use hpo.jax.org/app)**
Abnormal amyloid beta 40 isoform level 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 104300

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
To the best of my knowledge the findings related to amyloid beta described here are research findings but it is not the case that we can define a threshold for abnormal levels of this isoform that could be used as a standard clinical test (this is what is in scope for HPO). Sorry for the late response. Typically lab findings can be included in the HPO if they represent clinical tests. In this case, there is often a LOINC code. If this is actually the case, please re-contact.

**Comment by pnrobinson** [comment]
I can find only for 42
https://loinc.org/33203-1/
which I will add
